CMakeLists.txt

```diff
@@ -44,28 +43,22 @@ catkin_package(
     roscpp
     rospy
     message_generation
+    message_runtime
 )
 
 include_directories(
   ${catkin_INCLUDE_DIRS}
 )
+
 # Declare a C++ library
-set(SRCS
+set(ros_alarms_srcs
```

Comment:
convention seems to be ALL_CAPS for variables like these to differentiate from targets